dbr_test.go

```diff
@@ -88,6 +89,7 @@ func reset(t *testing.T, sess *Session) {
 		_, err := sess.Exec(v)
 		require.NoError(t, err)
 	}
+	sess.EventReceiver = &testTraceReceiver{}
```

Comment:
I don't think this is needed. `Open()` will already pass EventReceiver to all created sessions.

Reply:
Oh, I'm trying to reset the state inside the `testTraceReceiver{}` (which collects data about each call to the `TraceEventReceiver` interface for use in test assertions). Is that OK? I can add a comment.

Reply:
👍